integration_test/lit.cfg.py

```diff
@@ -149,5 +150,7 @@
 # Enable Python bindings tests if they're supported.
 if config.bindings_python_enabled:
   config.available_features.add('bindings_python')
+if config.bindings_tcl_enabled:
+  config.available_features.add('bindings_tcl')
```

Comment:
You might also need to add the location of `tclsh` to `tool_dirs` and `tclsh` to `tools`? Check out what happens if `config.verilator_path != ""`, for example.

Reply:
It works without this. Is there a reason why this should be done?

Reply:
I'm honestly not sure why we've been populating `tools` and `tool_dirs`. Perhaps there is some feature to query those lists that we aren't using. If it works without it, then I guess there's no reason to add it.